src/web/core/src/main/java/applicationContext.xml

```diff
@@ -293,6 +293,13 @@
     <property name="componentClass" value="org.geoserver.web.publish.RootLayerConfig" />
   </bean>
 
+  <!-- LayerGroup configuration panels -->
+  <bean id="httpLayerGroupConfigPanel" class="org.geoserver.web.publish.LayerGroupConfigurationPanelInfo">
```

Comment:
Would be good to have a test checking that the layer group caching panel shows up in the UI, and that submitting works (e.g. modify LayerGroupEditPageTest or LayerGroupPageTest)

Reply:
There were tests already in parent class. [Here](https://github.com/geoserver/geoserver/blob/4323399e06370758d8dec3b9022f01726b3390bb/src/web/core/src/test/java/org/geoserver/web/publish/HTTPLayerConfigTest.java#L68-L84)

Is this enough?

Reply:
Is the parent class testing the integration of the panel with layer groups? I want to see the application context declaration is correctly picked up, and working, against a layer group.